src/Libraries/GeometryColor/ImportHelpers.cs

```diff
@@ -40,22 +40,22 @@ public static IEnumerable<Geometry> ImportFromSATWithUnits(string filePath, [Def
         /// <param name="dynamoUnit">a forge unit length, if left null, sat file will be imported as unitless</param>
         /// <returns></returns>
         [AllowRankReduction]
-        public static IEnumerable<Geometry> ImportFromSATWithUnits(FileInfo file, [DefaultArgument("null")] DynamoUnits.Unit dynamoUnit)
+        public static IEnumerable<Geometry> ImportFromSATWithUnits(FileInfo file, DynamoUnits.Unit dynamoUnit)
         {
             var mm_per_unit = CalculateMillimeterPerUnit(dynamoUnit);
             return Geometry.ImportFromSAT(file, mm_per_unit);
         }
 
 
         /// <summary>
-        /// Imports geometry from SAB filepath. Set the dynamoUnit input to match how you are 
+        /// Imports geometry from SAB byte array. Set the dynamoUnit input to match how you are 
         /// interperting the other numbers in your Dynamo file.
         /// </summary>
         /// <param name="buffer">SAB byte array</param>
         /// <param name="dynamoUnit">a forge unit length, if left null, sat file will be imported as unitless</param>
         /// <returns></returns>
         [AllowRankReduction]
-        public static IEnumerable<Geometry> DeserializeFromSABWithUnits(byte[] buffer, [DefaultArgument("null")] DynamoUnits.Unit dynamoUnit)
+        public static IEnumerable<Geometry> DeserializeFromSABWithUnits(byte[] buffer, DynamoUnits.Unit dynamoUnit)
```

Comment:
Isn't the default for mmPerUnit decided as -1, in which case, there should be a corresponding default argument for `DynamoUnits.Unit`?

Reply:
it was not necessary because the node model already has a default, but I can add it back if you prefer both the internal and external method to have the same default.

Reply:
I'm guessing that these methods could still be used in Python, code blocks, C# code, etc. so might be good to add them back.